### DISCUSSION TITLE
MS Teams tab template in VS Code

### DISCUSSION THREAD

**Discussion by Fanttatraieste** [OfficeDev/microsoft-365-agents-toolkit#10519] [Q&A] [answered]
Hello, I saw that you have a template for react, and I want to integrate my angular app. Is it possible ? What should I do ? I tried more things then I can mention in order to integrate my angular app inside the basic tab template. Do you have any ideas, solutions ? Are there any docs that I can read about this topic ?

**Comment by therealjohn** [comment, marked as answer]
@Fanttatraieste 

We don't have a sample for this, but the closest thing we have is https://github.com/OfficeDev/TeamsFx/wiki/Make-your-project-compatible-with-Teams-Toolkit

That has info on each part of the toolkit you can onboard to and how to do it.

**Reply by Fanttatraieste** [reply]
Thank you ! It is extraordinary helpfull ! I've tried similar things on my own looking at the react template, typescript template and js template, but now with the guidance of this wiki, I'll succeed. Thank you very much !
